hedyorg/hedy
# [BUG] Improve error message level 5

**Fixes #3329**

Current main:
<img width="774" alt="image" src="https://user-images.githubusercontent.com/1003685/205283086-7721e8fe-58f3-455e-bd97-5aaccd03359b.png">

New:
<img width="757" alt="image" src="https://user-images.githubusercontent.com/1003685/205284340-41d7a5e0-5883-4669-8905-5ddc0600ab4b.png">

Always link the number of the issue or of the discussion that your PR concerns.
Tip, if you use the words fixes before the issue number in this description, the related issue will automatically close then the PR is merged! 

**How to test**

Try the code snippet above and see the right text is shown or run the added test.

**Checklist**
Done? Check if you have it all in place using this list:*
  

If you're unsure about any of these, don't hesitate to ask. We're here to help!
